LocalSettings.php

```diff
@@ -4785,9 +4783,8 @@
 	require_once '/srv/mediawiki/config/ExtensionMessageFiles.php';
 }
 
-// Last Stuff
-$wgConf = $wi->config;
-unset( $wi );
+// Don't need a global here
+// unset( $wi );
```

Comment:
This makes me think this PR is a draft. Feel free to correct me.

Reply:
Not really a "draft" but I did comment it out so when I pull to test I can verify the property outputs from that via eval. So its ready for testing, just that was done for testing it. It will be uncommented right before merging. Thanks!